tools/worldclim/raster_getdata.xml

```diff
@@ -0,0 +1,204 @@
+                <option value="no_plot">No</option>
+            </param>
+            <when value="yes_plot">
+                <param name="var_to_plot" type="select" label="Select what you want to visualize">
+                    <option value="1">[tmin|tmax|prec]:January</option>
+                    <option value="2">[tmin|tmax|prec]:February</option>
+                    <option value="3">[tmin|tmax|prec]:March</option>
+                    <option value="4">[tmin|tmax|prec]:April</option>
+                    <option value="5">[tmin|tmax|prec]:May</option>
+                    <option value="6">[tmin|tmax|prec]:June</option>
+                    <option value="7">[tmin|tmax|prec]:July</option>
+                    <option value="8">[tmin|tmax|prec]:August</option>
+                    <option value="9">[tmin|tmax|prec]:September</option>
+                    <option value="10">[tmin|tmax|prec]:October</option>
+                    <option value="11">[tmin|tmax|prec]:November</option>
+                    <option value="12">[tmin|tmax|prec]:December</option>
+                    <option value="bio1">bio:Annual Mean Temperature</option>
+                    <option value="bio2">bio:Mean Diurnal Range (Mean of monthly (max temp - min temp))</option>
+                    <option value="bio3">bio:Isothermality (BIO2/BIO7) (* 100)</option>
+                    <option value="bio4">bio:Temperature Seasonality (standard deviation *100)</option>
+                    <option value="bio5">bio:Max Temperature of Warmest Month</option>
+                    <option value="bio6">bio:Min Temperature of Coldest Month</option>
+                    <option value="bio7">bio:Temperature Annual Range (BIO5-BIO6)</option>
+                    <option value="bio8">bio:Mean Temperature of Wettest Quarter</option>
+                    <option value="bio9">bio:Mean Temperature of Driest Quarter</option>
+                    <option value="bio10">bio:Mean Temperature of Warmest Quarter</option>
+                    <option value="bio11">bio:Mean Temperature of Coldest Quarter</option>
+                    <option value="bio12">bio:Annual Precipitation</option>
+                    <option value="bio13">bio:Precipitation of Wettest Month</option>
+                    <option value="bio14">bio:Precipitation of Driest Month</option>
+                    <option value="bio15">bio:Precipitation Seasonality (Coefficient of Variation)</option>
+                    <option value="bio16">bio:Precipitation of Wettest Quarter</option>
+                    <option value="bio17">bio:Precipitation of Driest Quarter</option>
+                    <option value="bio18">bio:Precipitation of Warmest Quarter</option>
+                    <option value="bio19">bio:Precipitation of Coldest Quarter</option>
+                </param>
+
+                <param name="usr_country" type="select" label="Select a worldmap vision or a more precise region" >
```

Comment:
General question. Does such a param makes sense to be included in Galaxy main. Maybe even with a fancy world-map selector? Just keep in mind we can add cool fancy new params to Galaxy if this is needed and makes sense for communities.

Reply:
I'm not sure to understand, do you mean we can add a "d3js-like" or shiny visualisation ? It would be cool, but easily doable with raster file ?  This param is just a quick and easy way to offer a visualisation once the raster is loaded by the R script

Reply:
Not shiny, its needs to be JS only. But for example do you know that Galaxy has a color-picker param (https://docs.galaxyproject.org/en/latest/dev/schema.html#id36). If its needed we could probably develop a coordinate picker param or similar.

Reply:
I didn't knew about color-picker and just tested it, fancy indeed :) A coordinate-picker could have a lot of interest in GIS related tools. In this case though the select only get the country iso code needed in a R function

Reply:
Just pointing out ideas here :) Keep in mind this is possible and if you find a JS only widget that we could use we can integrate this directly into Galaxy :)

Reply:
Interesting ideas ! Thank you I will keep this in mind :)

Reply:
Never see this color-picker things too.... Very ineresting... For the future, if we consider a JS based fancy world-map selector, for sure we have to test things with [leaflet](https://leafletjs.com/)...